src/app/reservation/create-reservation/date-and-time-step/date-and-time-step.component.ts

```diff
@@ -50,8 +51,19 @@ export class DateAndTimeStepComponent {
   }
 
   protected addReservationDateAndTime(): void {
+    if (!this.selectedDate || !this.startTime || !this.endTime) {
+      this.toastr.error('Alle velden zijn verplicht!', 'Validatie Error');
+      return;
```

Comment:
je kan ook de toast returns scheelt 1 line

Reply:
kan niet want die is void